src/evaluate.py

```diff
@@ -153,6 +153,11 @@ def store_results(config, results_file, predictions_file, models, ids, y_pred, m
             config['task'] = 'alterations'
             config['alteration'] = alteration
 
+        if 'melspectrogram' in config['audio_rep']['type']:
```

Comment:
Is there a reason to do the check with `in` or could it also be done with `==`?

Reply:
We should also consider the case of VGGish features that are also spectrograms but using a different extractor. Right now these features are called `vggish` but I could rename them as `vggish_spectrograms` in other PR.